### PR TITLE
Fixing id not found errors

### PR DESCRIPTION
**Pull request by schwabts**
* `import` statements seem to need to be updated,
* my changes for using the sample in a multipage app might not be desired when explaining the underlying principle but this PR is essentially for showing the page's exact state in which I got the error `ID not found in layout`,
* markdown format of Mongo guide was more convenient for me.

### CODE REVIEW

File: Bootstrap/bootstrap_card.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+# Docs:     [Dash Bootstrap Components:](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/card/)
+#           [Dash HTML/CORE Components:](https://dash.plotly.com/dash-html-components)
+#
+
+import dash                                     # pip install dash
+import dash_bootstrap_components as dbc         # pip install dash-bootstrap-components
+from dash import dcc 
+from dash import html
+import plotly.express as px                     # # pip install pandas; pip install plotly express
 from dash.dependencies import Input, Output
```
Comment (Coding-with-Adam) [inline Bootstrap/bootstrap_card.py:11]:
```suggestion
#
from dash import Dash, dcc, html, Output, Input       # pip install dash
import dash_bootstrap_components as dbc               # pip install dash-bootstrap-components
import plotly.express as px                     # pip install pandas; pip install plotly express
```

File: Bootstrap/bootstrap_card.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
-import dash
-import dash_bootstrap_components as dbc
-import dash_html_components as html
-import dash_core_components as dcc
-import plotly.express as px
-from dash.dependencies import Input, Output
+# Video:    [Bootstrap with Cards - Dash Plotly](https://youtu.be/aEz1-72PKwc)
+# Docs:     [Dash Bootstrap Components:](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/card/)
+#           [Dash HTML/CORE Components:](https://dash.plotly.com/dash-html-components)
+#
```
Comment (Coding-with-Adam) [inline Bootstrap/bootstrap_card.py:4]:
```suggestion
```

File: Bootstrap/bootstrap_collapse2.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
-import dash
-import dash_bootstrap_components as dbc
-import dash_html_components as html
-import dash_core_components as dcc
+import dash                                     # pip install dash
+import dash_bootstrap_components as dbc         # pip install dash_bootstrap_components
+from dash import html
+from dash import dcc
 import plotly.express as px
 from dash.dependencies import Input, Output, State
```
Comment (Coding-with-Adam) [inline Bootstrap/bootstrap_collapse2.py:6]:
```suggestion
from dash import Dash, dcc, html, Input, Output, State        # pip install dash
import dash_bootstrap_components as dbc             # pip install dash_bootstrap_components
import plotly.express as px



```

File: Bootstrap/bootstrap_collapse2.py [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
-import dash_core_components as dcc
+from dash import Dash, dcc, html, Input, Output, State        # pip install dash
+import dash_bootstrap_components as dbc             # pip install dash_bootstrap_components
 import plotly.express as px
-from dash.dependencies import Input, Output, State
+
+
+
 
 import pandas as pd
```
Comment (Coding-with-Adam) [inline Bootstrap/bootstrap_collapse2.py:8]:
```suggestion
import pandas as pd




```

File: Bootstrap/bootstrap_intro.py [hunk 5]
```diff
@@ -1,7 +1,12 @@
...
+#           [Dash Bootstrap Themes:](https://dash-bootstrap-components.opensource.faculty.ai/docs/themes/)
+#
+
+import dash                                     # pip install dash
+import dash_bootstrap_components as dbc         # pip install dash_bootstrap_components
+from dash import html
+from dash import dcc
 from dash.dependencies import Input, Output
 import plotly.express as px
 import pandas as pd
```
Comment (Coding-with-Adam) [inline Bootstrap/bootstrap_intro.py:12]:
```suggestion
# Video:    [Introduction to Dash Bootstrap - Styling your App](https://youtu.be/vqVwpL4bGKY)
# Docs:     [Dash Bootstrap Components:](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/alert/)
#           [Dash Bootstrap Themes:](https://dash-bootstrap-components.opensource.faculty.ai/docs/themes/)
from dash import Dash, dcc, html, Input, Output         # pip install dash
import dash_bootstrap_components as dbc         # pip install dash_bootstrap_components
import plotly.express as px
import pandas as pd
```

File: Bootstrap/bootstrap_modal.py [hunk 6]
```diff
@@ -1,9 +1,10 @@
...
-import dash_core_components as dcc
+# Video:    [Bootstrap Alerts & Modals - Dash Plotly](https://youtu.be/X3OuhqS8ueM)
+# Docs:     [Dash Bootstrap Components:](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/alert/)
+#           [Dash Bootstrap Themes:](https://dash-bootstrap-components.opensource.faculty.ai/docs/themes/)
+#           [Dash HTML/CORE Components:](https://dash.plotly.com/dash-html-components)
+from dash import Dash, dcc, html, Input, Output, State           # pip install dash
+import dash_bootstrap_components as dbc         # pip install dash_bootstrap_components
 import plotly.express as px
-from dash.dependencies import Input, Output, State
 import pandas as pd
```
Comment (Coding-with-Adam) [inline Bootstrap/bootstrap_modal.py:9]:
```suggestion
```

File: Dash Components/BeforeAfter/before-after.py [hunk 7]
```diff
@@ -1,5 +1,8 @@
-import dash                              # pip install dash
-import dash_html_components as html
+# Video:    [Before After Image Slider - Dash Component](https://youtu.be/-0bqU94KGMw)
+#
+
+import dash                                     # pip install dash
+from dash import html
 from dash_extensions import BeforeAfter  # pip install dash-extensions==0.0.47 or higher
 import dash_bootstrap_components as dbc  # dash-bootstrap-components
```
Comment (Coding-with-Adam) [inline Dash Components/BeforeAfter/before-after.py:7]:
```suggestion
# Video:    [Before After Image Slider - Dash Component](https://youtu.be/-0bqU94KGMw)
from dash import Dash, html                                     # pip install dash
from dash_extensions import BeforeAfter  # pip install dash-extensions==0.0.47 or higher
import dash_bootstrap_components as dbc  # dash-bootstrap-components
```

File: Dash Components/Checklist/checkbox.py [hunk 8]
```diff
@@ -1,9 +1,13 @@
...
 
-import dash             
-import dash_core_components as dcc
-import dash_html_components as html
+import dash                                     # pip install dash
+from dash import dcc 
+from dash import html
 from dash.dependencies import Input, Output
 
 import plotly
```
Comment (Coding-with-Adam) [inline Dash Components/Checklist/checkbox.py:13]:
```suggestion
import dash                                     # pip install dash
from dash import dcc, html, Input, Output
```

File: Dash Components/Daq_Components/daq.py [hunk 9]
```diff
@@ -1,7 +1,11 @@
-import dash
+# Video:    [Introduction to Dash DAQ for manufacturing dashboards](https://youtu.be/t3cLkzJAUgo)
+# Docs:     [Dash DAQ](https://dash.plotly.com/dash-daq)
+#
+
+import dash                                     # pip install dash
+from dash import dcc 
+from dash import html
 from dash.dependencies import Input, Output
```
Comment (Coding-with-Adam) [inline Dash Components/Daq_Components/daq.py:8]:
```suggestion
# Video:    [Introduction to Dash DAQ for manufacturing dashboards](https://youtu.be/t3cLkzJAUgo)
# Docs:     [Dash DAQ](https://dash.plotly.com/dash-daq)
import dash                                     # pip install dash
from dash import dcc, html, Input, Output
```

File: Dash Components/DatePickerRange/datepicker.py [hunk 10]
```diff
@@ -1,8 +1,13 @@
...
 from datetime import datetime as dt
 import plotly.express as px
-import dash
-import dash_html_components as html
-import dash_core_components as dcc
+import dash                                     # pip install dash
+from dash import dcc 
+from dash import html
 from dash.dependencies import Input, Output
 import pandas as pd
```
Comment (Coding-with-Adam) [inline Dash Components/DatePickerRange/datepicker.py:13]:
```suggestion
import dash                                     # pip install dash
from dash import dcc, html, Input, Output
import pandas as pd

```

File: Dash Components/DatePickerRange/datepicker.py [hunk 11]
```diff
@@ -1,9 +1,12 @@
+# Video:    [DatePickerRange - Python Dash Plotly](https://youtu.be/5uwxoxaPD8M)
+# Docs:     [dcc.DatePickerRange](https://dash.plotly.com/dash-core-components/datepickerrange)
+#           [plotly.express.density_mapbox](https://plotly.com/python-api-reference/generated/plotly.express.density_mapbox.html#plotly.express.density_mapbox)
+#
+
```
Comment (Coding-with-Adam) [inline Dash Components/DatePickerRange/datepicker.py:5]:
```suggestion
```

File: Dash Components/DatePickerRange/datepicker.py [hunk 12]
```diff
@@ -1,9 +1,10 @@
...
+#           [plotly.express.density_mapbox](https://plotly.com/python-api-reference/generated/plotly.express.density_mapbox.html#plotly.express.density_mapbox)
 from datetime import datetime as dt
 import plotly.express as px
-import dash
-import dash_html_components as html
-import dash_core_components as dcc
-from dash.dependencies import Input, Output
+import dash                                     # pip install dash
+from dash import dcc, html, Input, Output
 import pandas as pd
```
Comment (Coding-with-Adam) [inline Dash Components/DatePickerRange/datepicker.py:9]:
```suggestion
```

File: Dash Components/Dropdown/dropdown.py [hunk 13]
```diff
@@ -1,10 +1,15 @@
+# Video:    [Dropdown Selector- Python Dash Plotly](https://youtu.be/UYH_dNSX1DM)
+# Docs:     [dcc.Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
+#
+
 import pandas as pd     #(version 1.0.0)
 import plotly           #(version 4.5.4) pip install plotly==4.5.4
 import plotly.express as px
```
Comment (Coding-with-Adam) [inline Dash Components/Dropdown/dropdown.py:8]:
```suggestion
#
import pandas as pd     #(version 1.0.0)
import plotly.express as px
```

File: Dash Components/Dropdown/dropdown.py [hunk 14]
```diff
@@ -1,10 +1,12 @@
...
-
 import dash             #(version 1.9.1) pip install dash==1.9.1
-import dash_core_components as dcc
-import dash_html_components as html
+from dash import dcc 
+from dash import html
+ 	
 from dash.dependencies import Input, Output
 
 app = dash.Dash(__name__)
```
Comment (Coding-with-Adam) [inline Dash Components/Dropdown/dropdown.py:12]:
```suggestion
from dash import dcc, html, Input, Output


app = dash.Dash(__name__)
```

File: Dash Components/Graph/dash-graph.py [hunk 15]
```diff
@@ -1,6 +1,10 @@
...
-import dash_html_components as html
+# Video:    [All about the Graph Component - Python Dash Plotly](https://youtu.be/G8r2BB3GFVY)
+# Docs:     [dcc.Graph](https://dash.plotly.com/dash-core-components/graph)
+#
+
+import dash                                     # pip install dash
+from dash import dcc 
+from dash import html
 from dash.dependencies import Output, Input
 import plotly.express as px
```
Comment (Coding-with-Adam) [inline Dash Components/Graph/dash-graph.py:10]:
```suggestion
import dash                                     # pip install dash
from dash import dcc, html, Output, Input
import plotly.express as px

```

File: Dash Components/Input_element/input_non_text.py [hunk 16]
```diff
@@ -1,6 +1,10 @@
...
+# Video:    [Input Box - Python Dash Plotly](https://youtu.be/VZ6IdRMc0RI)
+# Docs:     [dcc.Input](https://dash.plotly.com/dash-core-components/input)
+#
+
+import dash                                     # pip install dash
+from dash import dcc 
+from dash import html
 import pandas as pd
 import plotly.express as px     # (version 4.6.0)
 from dash.dependencies import Input, Output
```
Comment (Coding-with-Adam) [inline Dash Components/Input_element/input_non_text.py:10]:
```suggestion
import dash                                     # pip install dash
from dash import dcc, html, Input, Output 
import pandas as pd
import plotly.express as px     # (version 4.6.0)
```

File: Dash Components/Input_element/input_text.py [hunk 17]
```diff
@@ -1,6 +1,10 @@
...
-import dash_core_components as dcc
-import dash_html_components as html
+# Video:    [Input Box - Python Dash Plotly](https://youtu.be/VZ6IdRMc0RI)
+# Docs:     [dcc.Input](https://dash.plotly.com/dash-core-components/input)
+#
+
+import dash                                     # pip install dash
+from dash import dcc 
+from dash import html
 from dash.dependencies import Input, Output
```
Comment (Coding-with-Adam) [inline Dash Components/Input_element/input_text.py:8]:
```suggestion
import dash                                     # pip install dash
from dash import dcc, html, Input, Output
```

File: Dash Components/Interval/interval.py [hunk 18]
```diff
@@ -1,9 +1,14 @@
...
-import dash             
-import dash_core_components as dcc
-import dash_html_components as html
+	 
+import dash                                     # pip install dash
+from dash import dcc 
+from dash import html
+ 	 
 from dash.dependencies import Input, Output
 from dash.exceptions import PreventUpdate
```
Comment (Coding-with-Adam) [inline Dash Components/Interval/interval.py:13]:
```suggestion
import dash                                     # pip install dash
from dash import dcc, html, Input, Output
from dash.exceptions import PreventUpdate
```

File: Dash Components/Interval/interval.py [hunk 19]
```diff
@@ -1,10 +1,11 @@
+# Video: [The Dash Interval overview](https://youtu.be/OSFoBbNnPWk)
+# Docs:  [dcc.Interval](https://dash.plotly.com/dash-core-components/interval)
+#
+
 import plotly
```
Comment (Coding-with-Adam) [inline Dash Components/Interval/interval.py:5]:
```suggestion
```

File: Dash Components/Range_Slider/rangeslider.py [hunk 20]
```diff
@@ -1,7 +1,15 @@
...
-
+			  
+import dash                                     # pip install dash
+from dash import dcc 
+from dash import html
+import plotly.express as px                     # # pip install pandas; pip install plotly express			  
+			  
 import dash             #(version 1.8.0)
 import dash_core_components as dcc
 import dash_html_components as html
```
Comment (Coding-with-Adam) [inline Dash Components/Range_Slider/rangeslider.py:15]:
```suggestion  
import dash                                            # pip install dash
from dash import dcc, html
import plotly.express as px                     # # pip install pandas; pip install plotly express			  
```

File: Dash Components/Range_Slider/rangeslider.py [hunk 21]
```diff
@@ -1,10 +1,12 @@
+# Video:    [Range Slider -- Python Dash Plotly](https://youtu.be/Zvz2LpziQAs)
+# Docs:     [dcc.RangeSlider](https://dash.plotly.com/dash-core-components/rangeslider)
+#
+
 import pandas as pd     #(version 1.0.0)
 import plotly           #(version 4.5.0)
```
Comment (Coding-with-Adam) [inline Dash Components/Range_Slider/rangeslider.py:6]:
```suggestion
import pandas as pd     #(version 1.0.0)
```

File: Dash_and_Databases/MongoDB/Mongo-Guide.md [hunk 22]
```diff
@@ -1,2 +1,54 @@
-Click here to download the guide that I created to help you follow the tutorial.
-  https://drive.google.com/file/d/1Z94SArM9azTcykVdNxmPQPAXeLfNtjJJ/view?usp=sharing
+# Install Mongo and Build Local Server
```
Comment (Coding-with-Adam) [inline Dash_and_Databases/MongoDB/Mongo-Guide.md:1]:
```suggestion
See instruction below or download them here: 
https://drive.google.com/file/d/1Z94SArM9azTcykVdNxmPQPAXeLfNtjJJ/view?usp=sharing
# Install Mongo and Build Local Server
```

**Review by Coding-with-Adam — APPROVED** [review]
Thank you so much for your help. 
I modified the imports slightly and reduced empty spaces since I want the users to view the code lines as they saw them in the recorded video.

**Comment by Coding-with-Adam** [comment]
I've been looking at your sample in a multipage app, but I don't understand either why you're getting an ID not found. I'm sorry

**Comment by schwabts** [comment]
I just noticed you still kept the old import statements
```
import dash_core_components as dcc
import dash_html_components as html
```
in [mongo_dash_datatable.py](https://github.com/Coding-with-Adam/Dash-by-Plotly/blob/master/Dash_and_Databases/MongoDB/mongo_dash_datatable.py), anyway.

**Comment by Coding-with-Adam** [comment]
Thanks for catching that, @schwabts I just update that.

**Comment by mboiwahab** [comment]
Thanks Adam

**Pull request merged**